openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +484,64 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme, and the content type "application/x-www-form-urlencoded" and the accept header set to "application/oauth-authz-req+jwt". 
+
+The following parameters are defined: 
+
+`wallet_metadata`:
+: OPTIONAL. OPTIONAL. A String containing a JSON Object containing metadata parameters as defined in (#as_metadata_parameters). 
+
+`wallet_nonce`:
+: OPTIONAL. A String containing a value the Verifier MUST use in the `wallet_nonce` authorization parameter when creating the signed presentation request object. For example, a base64url encoded fresh, cryptographically random number with sufficient entropy.  
+
+If the Wallet wants the Verifier to encrypt the request object, it SHOULD use the `jwks` or `jwks_uri` parameter within `wallet_metadata` to pass the public key for the input to the key agreement. Other mechanisms to pass the encryption key can be used as well. 
+
+The following is a non-normative example of a request:
+
+```
+  POST /request HTTP/1.1
+    Host: client.example.org
+    Content-Type: application/x-www-form-urlencoded
+
+    wallet_metadata=%7B%22vp_formats_supported%22%3A%7B%22jwt_vc_json%22%3A%7B%22alg_values_supported
+    %22%3A%5B%22ES256K%22%2C%22ES384%22%5D%7D%2C%22jwt_vp_json%22%3A%7B%22alg_values_supported%22%3A%
+    5B%22ES256K%22%2C%22EdDSA%22%5D%7D%7D%7D&
+    wallet_nonce=qPmxiNFCR3QTm19POc8u
+```
+
+### Request URI Response
+
+The Request URI response MUST be an HTTP response with the content type "application/oauth-authz-req+jwt" and the body being a signed, optionally encrypted, request object as defined in [@RFC9101]. The request object MUST fulfill the requirements as defined in (#vp_token_request).
```

Comment:
Does it have to be always signed?

Reply:
I think it's worth considering allow a json content type for the response when using the new post mode, and hence deprecating the use of unsigned jws. It might be something to discuss after this is merged.

Reply:
filed issue #130